installer/templates/new/widgets/text/text.scss

```diff
@@ -22,11 +23,11 @@ $moreinfo-color:    rgba(255, 255, 255, 0.7);
   }
 
   .updated-at {
-    color: rgba(255, 255, 255, 0.7);
+    color: $updated-at-color;
   }
 
-
-  &.large h3 {
-    font-size: 65px;
+  & h3 {
```

Comment:
Unnecessary parent selector (&)

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#unnecessaryparentreference)